package.json

```diff
@@ -100,6 +101,6 @@
     "typescript": "^3.9.5"
   },
   "resolutions": {
-    "react-test-renderer": "^18.2.0"
+    "@types/react": "^16.14.2"
```

Comment:
Thanks, I thought we don't need it. But without this resolution, `@types/react` v17 will be installed since "@types/react-dom@<18.0.0" is depending on it.